lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -317,11 +317,11 @@ class SingleThreadQueryCountTestCase(ModuleStoreTestCase):
 
     @ddt.data(
         # old mongo with cache
-        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 12, 7),
-        (ModuleStoreEnum.Type.mongo, 50, 7, 5, 12, 7),
+        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 13, 8),
+        (ModuleStoreEnum.Type.mongo, 50, 7, 5, 13, 8),
         # split mongo: 3 queries, regardless of thread response size.
-        (ModuleStoreEnum.Type.split, 1, 3, 3, 12, 7),
-        (ModuleStoreEnum.Type.split, 50, 3, 3, 12, 7),
+        (ModuleStoreEnum.Type.split, 1, 3, 3, 13, 8),
+        (ModuleStoreEnum.Type.split, 50, 3, 3, 13, 8),
```

Comment:
I'd like to see a similar test for each of the reported use cases that shows the query counts going radically down. Unless I'm missing it, I only see tests for the behavior being correct. Having DDT tests like this makes it very clear when the number of queries accidentally goes up.